Instruments/config/DYJets_file.cfg

```diff
@@ -0,0 +1,105 @@
+[FILE dyjets]
+file_path: DYJetsToLL_M-50_ext1.root
+fileType: inclusive
+n_jet: 0 10
+n_bjet: 0 10
+n_ht: 0 10000
+
+[FILE dy1jets]
+file_path: DY1JetsToLL_M-50.root
```

Comment:
Does it works? Files are split in two folders: Full and p95, I think relative path should start from that point (at least for this production).